app/models/user.rb

```diff
@@ -6,10 +6,11 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   validates :display_name,
-  presence: true,
-  length: { maximum: 50 }
+    presence: true,
```

Comment:
I would ignore these comments, it is expecting the arguments to line up with the first one:

```ruby
validates :display_name,
              presence: true,
              length: { maximum: 50 }
```

I prefer the style you have where subsequent lines are indented two spaces.